scripts/dendrite_sytest.sh

```diff
@@ -45,6 +45,7 @@ TEST_STATUS=0
 mkdir -p /logs
 ./run-tests.pl -I Dendrite::Monolith -d $GOBIN -W /src/sytest-whitelist -O tap --all \
     --work-directory="/work" \
+    tests/30rooms/20typing.pl \
```

Comment:
this looks like it shouldn't be here. (why not just pass `tests/30rooms/20typing.pl` on the `docker run` commandline?)

Reply:
:D didn't know we passed all that through! Awesome thanks!